src/dev/run_check_core_api_changes.ts

```diff
@@ -18,43 +18,61 @@
  */
 
 import { ToolingLog } from '@kbn/dev-utils';
-import { Extractor, IExtractorConfig } from '@microsoft/api-extractor';
+import {
+  Extractor,
+  IConfigFile,
+  ExtractorLogLevel,
+  ExtractorConfig,
+  ExtractorResult,
+  ExtractorMessage,
+  ConsoleMessageId,
+} from '@microsoft/api-extractor';
 import chalk from 'chalk';
 import dedent from 'dedent';
 import execa from 'execa';
 import fs from 'fs';
+import path from 'path';
 import getopts from 'getopts';
 
-const apiExtractorConfig = (folder: string): IExtractorConfig => {
-  return {
+const apiExtractorConfig = (folder: string): ExtractorConfig => {
+  const config: IConfigFile = {
     compiler: {
-      configType: 'tsconfig',
-      rootFolder: '.',
+      tsconfigFilePath: '<projectFolder>/tsconfig.json',
     },
-    project: {
-      entryPointSourceFile: `target/types/${folder}/index.d.ts`,
-    },
-    apiReviewFile: {
+    projectFolder: path.resolve('./'),
+    mainEntryPointFilePath: `target/types/${folder}/index.d.ts`,
+    apiReport: {
       enabled: true,
-      apiReviewFolder: `./src/core/${folder}/`,
-      tempFolder: `./build/${folder}/`,
+      reportFileName: `${folder}.api.md`,
+      reportFolder: `<projectFolder>/src/core/${folder}/`,
+      reportTempFolder: `<projectFolder>/build/${folder}/`,
     },
-    apiJsonFile: {
+    docModel: {
       enabled: true,
-      outputFolder: `./build/${folder}`,
+      apiJsonFilePath: `./build/${folder}/${folder}.api.json`,
     },
-    dtsRollup: {
+    tsdocMetadata: {
       enabled: false,
-      trimming: true,
-      publishFolderForInternal: '',
-      publishFolderForBeta: '',
-      publishFolderForPublic: '',
-      mainDtsRollupPath: '',
     },
-    policies: {
-      namespaceSupport: 'permissive',
+    messages: {
+      extractorMessageReporting: {
+        default: {
+          logLevel: 'warning' as ExtractorLogLevel.Warning,
```

Comment:
ExtractorLogLevel is a const enum. When I use `ExtractorLogLevel.Warning` I get a runtime error  saying ExtractorLogLevel is undefined. If I just run typescript on the file, it seems to correctly inline this enum as:
```js
"default": {
                    logLevel: "warning" /* Warning */
                },
```

Reply:
This is a known caveat of https://babeljs.io/docs/en/babel-plugin-transform-typescript